CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Users using the cross device flow on desktop (instead of mobile) are now blocked from continuing
 - Internal: Removed unused development dependencies which had known vulnerabilities
 - Internal: Changed the way that blob/base64 files and images are rendered and passed through the system
+- Internal: Changed CSS units to be consistently `em`
```

Comment:
maybe worth explaining the caveat that it's still hooked to a pixel font sizes, since it requires further work.

Reply:
Good point, have done so 🙂